### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by HillaryOkello**
Worked on the mobile view of the website then did the tablet and desktop view. Made sure that the website is responsive on all screen types

### CODE REVIEW

**Comment by euqueme** [comment]
Hello guys :wave: 

I can't start the review because the README file is not descriptive please let's work on it and resubmit for a code review again thank you

- [ ] You should make sure that your repo looks professional before review. Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there).

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/latest\ 2.jpg);"></a>
+                <div>
+                <h4>Spotify’s song booster service offers exposure for royalties</h4>
+                  <small>Ivan Mehta ◽ 20 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/latest\ 3.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ] Please remove the `style` property from the element, let's do this in our CSS?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/voted.png);"></a>
+                <div>
+                <h4>Ransomware could interfere with 2020 elections</h4>
+                  <small>The Conversation ◽ 15 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/latest\ 2.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/twitter-engineer.png);"></a>
+                <div>
+                <h4>Engineer shares 5 tips on how to ace coding interviews</h4>
+                  <small>Yen Huang ◽ 13 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/voted.png);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/Tesla-supercharger-europe-germany-car-future-EV-charging-free-796x448.jpg);"></a>
+                <div>
+                <h4>Tesla hit with $14M fine over German battery recycling</h4>
+                  <small>Mathew Beedam ◽ 12 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/twitter-engineer.png);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/Degoo-Lifestyle-540x270.jpg);"></a>
+                <div>
+                <h4>Back up all your files into the cloud with this alternative</h4>
+                  <small>TNW Deals ◽ 10 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/Tesla-supercharger-europe-germany-car-future-EV-charging-free-796x448.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1049 @@
...
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/cloud.jpg);"></a>
+                <div>
+                <h4>How the cloud has helped businesses survive covid-19</h4>
+                  <small>Toni Sekinah ◽ 9 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/Degoo-Lifestyle-540x270.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1049 @@
...
+            <ul class="latest-col">
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/circuit.png);"></a>
+                <div>
+                <h4>No one but Nintendo could’ve made ‘Mario Kart Live:</h4>
+                  <small>Callum Booth ◽ 5 minutes ago</small>
+                </div>
+              </li>
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/cloud.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1049 @@
...
+            </div>
+        </div>
+        <div class="latest">
+          <section class="latest-section">
+            <h2>
+              <a href="#" class="lat">Latest news</a>
+            </h2>
+            <ul class="latest-col">
+              <li>
+                <a href="#" class="back" style="background-image: url(./images/circuit.png);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1049 @@
...
+              </div>
+            </div>
+          </section>
+        </div>
+        <section class="middle">
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Hard Fork</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/Hardfork.jpg);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Apps</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/Messenger-Image.jpg);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 11]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Gears</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/gear.png);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 12]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Tech</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/card-v-card-bank-account-multiplayer-game.png);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 13]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Creative</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/creative.jpg);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 14]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Podium</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/google-logo-podium-368x276.png);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]

File: index.html [hunk 15]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Insight</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/Netflix-Blur-vibration-haptic-feedback-project-rumble-pak-368x276.jpg);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 16]
```diff
@@ -0,0 +1,1049 @@
...
+                  Elon ‘own no home’ Musk sells Gene Wilder’s old pad — 5 properties to go
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Distract</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/Distract.jpeg);">
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 17]
```diff
@@ -0,0 +1,1049 @@
...
+          </div>
+        </section>
+
+        <section class="deals">
+          <h2>
+            <a href="#" class="lat">Latest deals</a>
+          </h2>
+          <ul class="latest-col">
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals1.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 18]
```diff
@@ -0,0 +1,1049 @@
...
+              <div>
+              <h4>Password Boss Premium: Lifetime Subscription</h4>
+                <small class="small-cross">$499</small><small>$34.99</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals2.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 19]
```diff
@@ -0,0 +1,1049 @@
...
+              <div>
+              <h4>Hushed Private Phone Line: Lifetime Subscription</h4>
+                <small class="small-cross">$150</small><small>$19.99</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals3.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 20]
```diff
@@ -0,0 +1,1049 @@
...
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals4.jpg);"></a>
+              <div>
+              <h4>TourBox: The Ultimate Controller for Creators</h4>
+                <small class="small-cross">$169</small><small>$149.99</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals5.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 21]
```diff
@@ -0,0 +1,1049 @@
...
+              <div>
+              <h4>XSplit VCam: Lifetime Subscription (Windows)</h4>
+                <small class="small-cross">$49</small><small>$19.99</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals6.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 22]
```diff
@@ -0,0 +1,1049 @@
...
+              <div>
+              <h4>StackSkills Unlimited: Lifetime Access</h4>
+                <small class="small-cross">$1495</small><small>$59</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals7.jpg);"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

File: index.html [hunk 23]
```diff
@@ -0,0 +1,1049 @@
...
+              <div>
+              <h4>The 2020 Adobe Graphic Design Certification School</h4>
+                <small class="small-cross">$1497</small><small>$49</small>
+              </div>
+            </li>
+            <li>
+              <a href="#" class="back" style="background-image: url(./images/deals8.jpg"></a>
```
Comment (MuriloRoque) [inline index.html]: - [ ]  Please remove the style property from the element, let's do this in our CSS?

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes Requested 🔴 

### Hello @HillaryOkello and @Honda99 👋 

### You have started really well, congratulations 👏 

### Please, follow my suggestions so you can improve your project:

## General

- [ ] Please fix your Web hint Linter errors.

- [ ] Your CSS code is repeating a lot of properties, which is considered bad practice since your code gets longer and harder to read. Consider creating classes for the properties you are repeating the most. For example: Create a class called `d-flex`, with the property `display: flex;` and apply this class to all of the elements that are "flexed". Below you can find the number of times you are repeating some properties:

      26 |   display: flex;
      11 |   list-style: none;
      10 |   width: 100%;
       9 |   text-decoration: none;

- [ ] You are using a lot of `!important`, which is also considered a bad practice, you could get lost in your code if you wanted to come back to this project and modify it, so please refactor your code and remove all of the `!important` properties, I know you can do it 💪 

- [ ] I added some comments in your code about another bad practice, which is adding inline-styling. Please refactor those to your CSS, so the styling is only in there 👍 

## Page Structure

**General**

- [ ] There's a horizontal scroll in your page, there must be some element messing up with the width of your page, please fix this so your users have a better experience when visualizing your page 😉 

horizontal


**Header**

***Big Screens***

- [ ] The TNW logo in the header only appears on **medium** screens, you are showing the logo on all screen sizes, please remove it from the **big** screens.

- [ ] The navigation links have a lighter color, yours are too dark when compared with the original.

**Yours:**

Yours


**Original:**

Original


***Medium Screens***

- [ ] The top navigation links should be present on this screen size, yours are vanishing when switching from **big screens** to **medium**.

**Yours:**

Yours


**Original:**

Original


**Three Images Section**

***Big Screens***

- [ ] There's a TNW logo inside of the big image to the left on the original page, please try adding it there.

- [ ] The font sizes of the text inside of the images on your page are too small compared to the original, please try following the same styling here too 😄 

**Yours:**

Yours


**Original:**

Original


***Small Screens***

- [ ] Part of the top image is being hidden behind the header, please fix this issue.

**Yours:**
Yours


**Original:**

Original


**Latest News Section**

***Big Screens***

- [ ] The images in the second column are positioned too far down compared to the other ones, please try aligning them.

- [ ] This section is too close to the section below it, please try adding some margin between them.

**Yours:**

Yours


**Original:**

Original


**Latest Funding Section**

***Big Screens***

- [ ] The title of this section on your page should be aligned to the far left, like in the original.

- [ ] You should have 4 article columns in this section, not 3.

**Yours:**

Yours


**Original:**

Original


***Medium Screens***

- [ ] The links that were aligned to the right on **big screens** should now be aligned below the title on **medium screens**, like the original.

**Yours:**

Yours


**Original:**

Original


**Hard Fork, Apps, Gear Section**

***Big screens***

- [ ] This section is not aligned to the middle, like the original.

**Yours:**

Yours


**Original:**

Original

## Summary

- The code is good overall, there are just some minor issues to fix 😄 
- Your repository is very professional, keep up the good work 💪 
- There are only some issues to solve and then we can move on 🚀 

Kindly make the requested changes and submit for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi team :wave: 

 Good work and great effort so far  :slightly_smiling_face: 

However, there are some designs that need to be edited, on your webpage.

### Header

Medium Screens
 
The top navigation links should be present on this screen size, yours are vanishing when switching from big screens to medium. Just as the last TSE pointed out.

### Latest News Section

Big Screens

 The images in the third column are not positioned properly. Compare them with the original website and make the necessary changes.

### Apps, Gears & Tech section

Medium screen 

@media >= 768px, Apps, Gear, Tech should be a grid with three columns; yours is a single column kindly reference the [main](https://thenextweb.com/) website.

[**Observation**] Please take your time to compare the [TNW](https://thenextweb.com/) with the **cloned** site you have designed. View it using chrome developer tools at a lower percentage, like 50%.  You would have a better perspective of the things you need to modify.

#### That is it, make the necessary changes, then submit another form for a code review.

If you need clarifications, do reach out on slack.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama```

File: index.html [hunk 24]
```diff
@@ -0,0 +1,1052 @@
...
+                </ul>
+              </div>
+            </div>
+          </section>
+        <section class="middle">
+          <div class="middle-1">
+            <h2>
+              <a href="#" class="lat">Hard Fork</a>
+            </h2>
+            <div class="grid" style="background-image: url(./images/Hardfork.jpg);">
```
Comment (uche-inyama) [inline index.html]: Please, do remove all the inline style you have in your HTML file

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @HillaryOkello :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**